Bugzilla/Elastic/Indexer.pm

```diff
@@ -238,7 +181,6 @@ sub _bulk_load_ids {
     }
 
     my $total = 0;
-    use Time::HiRes;
```

Comment:
No worries, just log it in the commit message or make it a separate commit. I don't mind cleanup, I just mind *unlogged* cleanup :)